sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -51,90 +51,114 @@ For detailed samples please review the samples provided.
-          << std::endl;
+<!-- @insert_snippet: SecretSample1UpdateSecretProperties -->
+```cpp
+    // change one of the properties
+    secret.Properties.ContentType = "my content";
+    // update the secret
+    KeyVaultSecret updatedSecret = secretClient.UpdateSecretProperties(secret.Properties).Value;
+    std::cout << "Secret's content type is now " << updatedSecret.Properties.ContentType.Value()
+              << std::endl;
 ```
 
 ### Delete a secret
 
 Delete an existing secret.
 
-```cpp Snippet:SecretSample1DeleteSecret
-// start deleting the secret
-DeleteSecretOperation operation = secretClient.StartDeleteSecret(secret.Name);
+<!-- @insert_snippet: SecretSample1DeleteSecret -->
+```cpp
+    // start deleting the secret
+    DeleteSecretOperation operation = secretClient.StartDeleteSecret(secret.Name);
+
+    // You only need to wait for completion if you want to purge or recover the secret.
+    // The duration of the delete operation might vary
+    // in case returns too fast increase the timeout value
+    operation.PollUntilDone(20s);
+
+    // purge the deleted secret
+    secretClient.PurgeDeletedSecret(secret.Name);
 ```
 
 ### Delete and purge a secret
 
 Delete and Purge a secret.
 
-```cpp Snippet:SecretSample1DeleteSecret
-// start deleting the secret
-DeleteSecretOperation operation = secretClient.StartDeleteSecret(secret.Name);
-// You only need to wait for completion if you want to purge or recover the secret.
-operation.PollUntilDone(std::chrono::milliseconds(2000));
-// purge the deleted secret
-secretClient.PurgeDeletedSecret(secret.Name);
+<!-- @insert_snippet: SecretSample1DeleteSecret -->
+```cpp
+    // start deleting the secret
+    DeleteSecretOperation operation = secretClient.StartDeleteSecret(secret.Name);
+
+    // You only need to wait for completion if you want to purge or recover the secret.
+    // The duration of the delete operation might vary
+    // in case returns too fast increase the timeout value
+    operation.PollUntilDone(20s);
+
+    // purge the deleted secret
+    secretClient.PurgeDeletedSecret(secret.Name);
 ```
 
 ### List Secrets
 
 List all the secrets in keyvault. 
 
-```cpp Snippet:SecretSample4ListAllSecrets
- // get properties of secrets
-for (auto secrets = secretClient.GetPropertiesOfSecrets(); secrets.HasPage(); secrets.MoveToNextPage())
-{ // go through every secret of each page returned
-  for (auto const& secret : secrets.Items)
-  {
-    std::cout << "Found Secret with name: " << secret.Name << std::endl;
-  }
-}
+<!-- @insert_snippet: SecretSample4ListAllSecrets -->
+```cpp
+    // get all the versions of a secret
+    for (auto secretsVersion = secretClient.GetPropertiesOfSecretsVersions(secret1.Name);
+         secretsVersion.HasPage();
+         secretsVersion.MoveToNextPage())
+    { // go through each version of the secret
+      // the number of results returned for in a  page is not guaranteed
```

Comment:
There's an extra space after 'in a '.  Is that wrapping issues as the snippet moves and is reformated?

Reply:
There is no reformatting of snippets. This is exactly what is in the source for the snippet.